src/socket.rs

```diff
@@ -369,6 +369,47 @@ impl Socket {
         self.inner.set_ttl(ttl)
     }
 
+    /// Gets the value for the `SO_MARK` option on this socket.
```

Comment:
https://github.com/rust-lang/socket2-rs/pull/132#discussion_r525316756 is not resolved. Remove the  `(set_)mark` functions here and move them to the `sys/unix.rs` file please. The same way `Socket::accept4` is exposed.

Reply:
I _think_ I've done that now. Let me know if it looks any better.